zeebe/broker/src/main/java/io/camunda/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -490,7 +490,18 @@ private void becomeLive() {
   private void startActiveExportingMode() {
     for (final ExporterContainer container : containers) {
       container.initMetadata();
-      container.openExporter();
+      new BackOffRetryStrategy(actor, Duration.ofSeconds(10))
+          .runWithRetry(
+              () -> {
+                try {
+                  container.openExporter();
+                  return true;
+                } catch (final Exception e) {
+                  LOG.error("Failed to open exporter '{}'. Retrying...", container.getId(), e);
+                  return false;
+                }
+              },
+              this::isClosed);
```

Comment:
:x: As we discussed before, we have to wait until all exporters are opened before we can proceed with the next step. With the current code, we might start exporting before the open is completed. This can lead to undefined behavior. 

Please also add a test to verify this once it is fixed.

Reply:
done